src/ui/public/inspector/inspector.tsx

```diff
@@ -62,14 +73,12 @@ function open(adapters: Adapters, options: InspectorOptions = {}): InspectorSess
 
   // Don't open inspector if there are no views available for the passed adapters
   if (!views || views.length === 0) {
-    throw new Error(`Tried to open an inspector without views being available.
-      Make sure to call Inspector.isAvailable() with the same adapters before to check
-      if an inspector can be shown.`);
+    throw new Error(openErrorMessage);
```

Comment:
This is a developer facing error only, since it should never be shown when using the API correctly, thus we don't need/should translate this.